fe1-web/src/__tests__/utils/TestUtils.ts

```diff
@@ -75,3 +78,61 @@ export const mockJsonRequest: Partial<JsonRpcRequest> = {
 };
 
 export const mockJsonResponse: Partial<JsonRpcResponse> = { id: 0, result: [] };
+
+// MOCK Transactions
+
+export const mockTransactionValue = 100;
+export const mockKPHash = Hash.fromPublicKey(mockKeyPair.publicKey);
+export const mockCBSig =
+  '1E7yPEY7SwZ1NNxi_jN85-IvThR7GzUFs8-Lfwao44waoCf-qj5xLKnpzsc8yP0cFr5UNOIzsgKUYaSzv1gpDQ==';
+export const mockCBHash = 'FhlMNTEOqOzkKbe8RH00fmF-Op0S_ipowEn0nj402Ts=';
+export const mockCoinbaseTransactionJSON: TransactionJSON = {
+  version: 1,
+  inputs: [
+    {
+      tx_out_hash: STRINGS.coinbase_hash,
+      tx_out_index: 0,
+      script: {
+        type: 'Pay-to-Pubkey-Hash',
```

Comment:
I know the google doc is out of sync, but all 3 other codebases and `protocol/` use `'P2PKH'`. There will be compatibility issues if we don't fix it here.

Reply:
I'll change that, but when changing the protocol the google doc should also be updated